test/test_client.py

```diff
@@ -0,0 +1,158 @@
+import json
+
+import pytest
+import app
+import requests
+from dotenv import load_dotenv, find_dotenv
+
+
+@pytest.fixture
+def client():
+    # Use our test integration config instead of the 'real' version
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+
+    # Create the new app.
+    test_app = app.create_app()
+    #
+    #     # Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+
+
+class MockResponse(object):
+    def __init__(self, status_code, body_content):
+        self._status_code = status_code
+        self._body_content = body_content
+
+    def json(self):
+        cards_with_json = json.loads(json.dumps(self._body_content))
+        cards = []
+        for card_json in cards_with_json:
+            cards.append(json.loads(card_json))
+        return cards
```

Comment:
Line 29 could just be `cards_with_json = self._body_content`

But better than that, this whole thing could just be `return self._body_content` if you remove the `'''` around the value of `generate_one_card_json`.

I don't see any value to the extra complexity.

Reply:
I might be missing something, but generate_one_card_json is a string and if i remove ``` it complains that is not valid